### PR TITLE
[stable/signalsciences] updated for new signalsciences version and best practices

### PR DESCRIPTION
**Pull request by jmymy**
@cpanato 
#### What this PR does / why we need it:
Brings this chart in line with official documentation from SignalSciences, as well as bumping to the official container.
https://docs.signalsciences.net/install-guides/kubernetes/kubernetes-agent-module/

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [X] Chart Version bumped
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by jmymy** [comment]
happy to be added as a maintainer.

File: stable/signalsciences/Chart.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ keywords:
 - firewall
 - waf
 sources:
-- https://github.com/marccampbell/docker-sigsci
+- https://github.com/helm/charts/tree/master/stable/signalsciences
```
Comment (cpanato) [inline stable/signalsciences/Chart.yaml]: this should points where you can find the code source, not the helm chart

**Review by cpanato — CHANGES_REQUESTED** [review]

File: stable/signalsciences/Chart.yaml (hunk 1 above)
Comment (jmymy) [reply stable/signalsciences/Chart.yaml]: i've updated to their github. The dockerfile for this container is not public yet.

**Comment by jmymy** [comment]
@cpanato how do the changes look? Thanks

**Review by cpanato — APPROVED** [review]
/approve
/lgtm

**Pull request merged**